builtin/mainmenu/dlg_delete_world.lua

```diff
@@ -17,39 +17,36 @@
 
 
 local function delete_world_formspec(dialogdata)
-
 	local retval =
-		"size[11.5,4.5,true]" ..
-		"label[2,2;" ..
+		"size[10,2.5,true]" ..
+		"label[0.5,0.5;" ..
 		fgettext("Delete World \"$1\"?", dialogdata.delete_name) .. "]" ..
-		"button[3.25,3.5;2.5,0.5;world_delete_confirm;" .. fgettext("Delete") .. "]" ..
-		"button[5.75,3.5;2.5,0.5;world_delete_cancel;" .. fgettext("Cancel") .. "]"
+		"button[0.5,1.5;2.5,0.5;world_delete_confirm;" .. fgettext("Delete") .. "]" ..
+		"button[7.0,1.5;2.5,0.5;world_delete_cancel;" .. fgettext("Cancel") .. "]"
 	return retval
 end
 
 local function delete_world_buttonhandler(this, fields)
 	if fields["world_delete_confirm"] then
-
 		if this.data.delete_index > 0 and
-			this.data.delete_index <= #menudata.worldlist:get_raw_list() then
+				this.data.delete_index <= #menudata.worldlist:get_raw_list() then
 			core.delete_world(this.data.delete_index)
 			menudata.worldlist:refresh()
+			this:delete()
```

Comment:
Please could you explain this change to me?

Reply:
I was adding the text field for confirmation, and I need to move it there so that it will only delete world and close dialog when the inputed name is same as the world name.

Probably I should revert that to original position, since the confirmation text has canceled.

Reply:
I was adding the text field for confirmation, and I need to move it there so that it will only delete the world and close the dialog when the inputted name is same as the world name.

Probably I should revert that to original position, since the confirmation text has canceled.

Reply:
Yes please, or we can do that on merge.

Reply:
I'll correct that on merge later if not done by then.

Reply:
Please do. Then i expect this will be approved and merged.

Reply:
Will do ASAP.

Reply:
Done.